index.js

```diff
@@ -0,0 +1,120 @@
+const allTripsUrl = " https://ada-backtrek-api.herokuapp.com/trips";
+
+const singleTripUrl = "https://ada-backtrek-api.herokuapp.com/trips/";
+
+const reserveUrl = "https://ada-backtrek-api.herokuapp.com/trips/1/reservations";
+
+const reportStatus = (message) => {
+  $('#status-message').html(message);
+};
+
+const reportError = (message, errors) => {
+  let content = `<p>${message}</p>`
+  content += "<ul>";
+  for (const field in errors) {
+    for (const problem of errors[field]) {
+      content += `<li>${field}: ${problem}</li>`;
+    }
+  }
+  content += "</ul>";
+  reportStatus(content);
+};
+
+const loadTrips = function loadTrips() {
+  let tripList = $('#trip-list');
+
+  axios.get(allTripsUrl)
```

Comment:
You don’t empty out the element holding your trip list before you load it, so if you click on “show trips” multiple times, the trips get added multiple times